src/test/java/org/fisco/bcos/channel/test/rpc/StartGroup.java

```diff
@@ -0,0 +1,50 @@
+package org.fisco.bcos.channel.test.rpc;
+
+import org.fisco.bcos.channel.client.Service;
+import org.fisco.bcos.web3j.protocol.Web3j;
+import org.fisco.bcos.web3j.protocol.channel.ChannelEthereumService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class StartGroup {
+
+    private static Logger logger = LoggerFactory.getLogger(GenerateGroup.class);
+
+    public static void Usage() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'Usage' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732338)